tests/ci/azure_pipeline_test/release_pipeline.yml

```diff
@@ -139,49 +139,6 @@ jobs:
   - release_nightly_linux_spark
   condition: succeeded()
   steps:
-    # Create archives with complete source code included
-  - task: ArchiveFiles@2
-    displayName: Create zip archive of reco_utils
-    condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/'))
-    inputs:
-      rootFolderOrFile: $(Build.SourcesDirectory)/reco_utils
-      includeRootFolder: false
-      archiveType: zip
-      archiveFile: '$(Build.ArtifactStagingDirectory)/archives/Recommenders-reco_utils_code.zip'
-      replaceExistingArchive: true
-
-  - task: ArchiveFiles@2
-    displayName: Create tar.gz archive of reco_utils
```

Comment:
We want to keep the tar.gz because we publish both this and the wheel on PyPI.

Reply:
running python setup.py sdist bdist_wheel creates both the wheel and a tarball, you then just need to upload everything in the dist folder that is created, no need to externally create a different tarball